draft-ietf-quic-transport.md

```diff
@@ -2642,14 +2642,14 @@ alternative services {{?ALTSVC=RFC7838}}.  Information that might allow correct
 routing of packets across multiple network paths will also allow activity on
 those paths to be linked by entities other than the peer.
 
-A client might wish to reduce linkability by employing a new connection ID and
-source UDP port when sending traffic after a period of inactivity.  Changing the
-UDP port from which it sends packets at the same time might cause the packet to
-appear as a connection migration. This ensures that the mechanisms that support
-migration are exercised even for clients that do not experience NAT rebindings
-or genuine migrations.  Changing port number can cause a peer to reset its
-congestion control state (see {{migration-cc}}), so the port SHOULD only be
-changed infrequently.
+A client might wish to reduce linkability by switching to a new connection ID,
+source UDP port, or IP address (see {{?RFC4941}}) when sending traffic after a
+period of inactivity.  Changing the UDP port from which it sends packets at the
+same time might cause the packet to appear as a connection migration. This
+ensures that the mechanisms that support migration are exercised even for
+clients that do not experience NAT rebindings or genuine migrations.  Changing
+port number can cause a peer to reset its congestion control state (see
+{{migration-cc}}), so the port SHOULD only be changed infrequently.
```

Comment:
You need to include "IP address" in subsequent sentences. I started doing that and found myself rewording more than just that.

```suggestion
A client can reduce linkability by switching to a new connection ID, changing
its source IP address (see {{?RFC4941}}), or changing its source port when
sending traffic after a period of inactivity. Changing a client's source IP
address or source port might cause the server to detect a connection migration,
ensuring that mechanisms that support migration are exercised even when clients
do not experience NAT rebindings or genuine migrations. The source IP address or
port number SHOULD only be changed infrequently at a client, since every change
can cause the server to reset its congestion control state (see
{{migration-cc}}).
```

Reply:
How about:

```suggestion
A client might wish to reduce linkability by switching to a new connection ID,
source UDP port, or IP address (see {{?RFC4941}}) when sending traffic after a
period of inactivity.  Changing the address from which it sends packets at the
same time might cause the packet to appear as a connection migration. This
ensures that the mechanisms that support migration are exercised even for
clients that do not experience NAT rebindings or genuine migrations.  Changing
address can cause a peer to reset its congestion control state (see
{{migration-cc}}), so addresses SHOULD only be changed infrequently.
```